Traceback (most recent call last):
  File "baseline.py", line 78, in <module>
    trainer = pl.Trainer(accelerator='auto',
  File "/lustre/home/acct-aemwx/aemwx-user1/.local/lib/python3.8/site-packages/pytorch_lightning/utilities/argparse.py", line 339, in insert_env_defaults
    return fn(self, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'distributed_backend'